plugins/ruby-foreman-omaha/debian/control

```diff
@@ -0,0 +1,14 @@
+Source: ruby-foreman-omaha
+Section: ruby
+Priority: extra
+Maintainer: Lennart Weller <lennart.weller@hansemerkur.de>
+Build-Depends: debhelper (>= 8.0.0), foreman (>= 1.18.0~rc1), git
```

Comment:
This needs `foreman-sqlite3` as well. Possibly foreman-assets

Reply:
Yeah sorry about the delay here. I'll get back on it in a bit.